dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessor.java

```diff
@@ -78,15 +83,29 @@ public void bindTo(StatementExecution cs, int ind) throws SQLException{
         this.cs=cs;
         this.index=ind;    
         if (direction != INPUT){
-            cs.registerOutParameter(ind, getSqlType());
+            cs.registerOutParameter(ind, getSqlType(), direction == RETURN_VALUE);
         }
     }
 
+    private Object transformValue(Object value) throws SQLException {
```

Comment:
And if `transformDbfitToJdbcType` (I think my preferred one) then `...Dbfit...` or `...DbFit...`?